companyweb_base/views/companyweb_partner_view.xml

```diff
@@ -0,0 +1,139 @@
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_equityCapital"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_addedValue"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_turnover"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_result"
+                                attrs="{'invisible':[('cweb_balance_year_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_warnings"
+                                attrs="{'invisible':[('cweb_warnings_enable', '!=', True )]}"
+                            />
+                                <field
+                                name="cweb_url"
+                                attrs="{'invisible':[('cweb_url_enable', '!=', True )]}"
+                            />
+
+                                <field
+                                name="cweb_vat_liable"
+                                attrs="{'invisible':[('cweb_vat_liable_enable', '!=', True )]}"
+                            />
+                            </group> <!-- end of group financial -->
+                            <group name="other">
+                                <field
+                                name="cweb_image"
+                                attrs="{'invisible':[('cweb_score_enable', '!=', True )]}"
+                            />
+                            </group> <!-- end of group other -->
+                        </group> <!-- end of group bottom-->
+                    </page>
+                </notebook>
+            </field>
+        </record>
+
+        <record id="companyweb_partner_view_button" model="ir.ui.view">
+            <field name="name">testxbo</field>
+            <field name="model">res.partner</field>
+            <field name="inherit_id" ref="base.view_partner_form" />
+            <field name="arch" type="xml">
+                <field name="vat" position="after">
+                    <button
+                    name="button_companyweb"
+                    string="Companyweb..."
+                    type="object"
+                    class="oe_inline"
+                    icon="https://www.companyweb.be/images/logo/logo-en.svg"
+                    attrs="{'invisible': ['|','|',('vat','&lt;','BE'),('vat','>=','BF'),('vat','=',False)]}"
```

Comment:
Better test on contry_id than the VAT prefix ?

Reply:
Also the button should only be shown for partners which are company (is_company=True)

Reply:
> Better test on contry_id than the VAT prefix ?

This is the attrs from V8, i can change this condition but a company can a company based outside of Belgium have a Belgian's VAT ?

Reply:
> is_company=True

Done, do you think i should also to that for the tab with companyweb info ?

Reply:
Actually what triggered this remark is the <BE|>=BF which is a bit of a hack. Better do a computed field that says if the companyweb button is visible.

Regarding the tab, I'd say it should be visible only if any of the cweb_*_enabled field is true. This can be done with a computed field.

Reply:
done